tests/profiling_v2/collector/test_memalloc.py

```diff
@@ -213,16 +226,50 @@ def get_tracemalloc_stats_per_func(stats, funcs):
         source_to_func[str(file) + str(line)] = f.__name__
 
     actual_sizes = {}
+    actual_counts = {}
     for stat in stats:
         f = stat.traceback[0]
         key = f.filename + str(f.lineno)
         if key in source_to_func:
-            actual_sizes[source_to_func[key]] = stat.size
-    return actual_sizes
+            func_name = source_to_func[key]
+            actual_sizes[func_name] = stat.size
+            actual_counts[func_name] = stat.count
+    return actual_sizes, actual_counts
+
+
+def test_unified_profiler_allocation_samples():
```

Comment:
I think at this point `test_memory_collector_allocation_accuracy_with_tracemalloc` does what this test does ( check that we get some non-in-use samples) plus more, so we could delete this test without losing coverage. WDYT?